OpenSim/Examples/ExampleHopperDevice/buildTestbedModel.cpp

```diff
@@ -1,7 +1,5 @@
-#ifndef _helperMethods_h_
-#define _helperMethods_h_
 /* -------------------------------------------------------------------------- *
- *                         OpenSim:  helperMethods.h                          *
+ *                   OpenSim:  buildTestbedModel.cpp                          *
```

Comment:
I'm sure someone had a reason and there was probably a discussion about it. I don't think it's important. Given the current contents of the file, I don't think it should be a header file.